cmd/ipfs/util/ulimit.go

```diff
@@ -0,0 +1,30 @@
+package util
+
+import (
+	"os"
+	"strconv"
+
+	"github.com/prometheus/common/log"
```

Comment:
Import `logging "github.com/ipfs/go-logging"` and then do a global `var log = logging.Logger("ulimit")`